ext/Rakefile

```diff
@@ -16,9 +16,9 @@ task :default => :clean do
     require "mini_portile2"
     recipe = MiniPortile.new("librdkafka", Rdkafka::LIBRDKAFKA_VERSION)
 
-    # Use default homebrew openssl if we're on mac and the directory exists
-    # and each of flags is not empty
-    if recipe.host&.include?("darwin") && system("which brew &> /dev/null") && Dir.exist?("#{homebrew_prefix = %x(brew --prefix openssl).strip}")
+    # Use default homebrew openssl if we're on mac and the directory exists, is not using nix-prepared libraries
+    # and each of flags is not already set
+    if recipe.host&.include?("darwin") && system("which brew &> /dev/null") && Dir.exist?("#{homebrew_prefix = %x(brew --prefix openssl).strip}") && !ENV["NIX_LDFLAGS"]
```

Comment:
can you use `key?` instead of the `!ENV` syntax pretty plz? :pray:

Reply:
I assume you want the other two checks to be changed to.

Reply:
@secobarbital if you want but it is not required. It's ok to change it here only as well if it's too much hustle for you. Thank you